src/CloudFlareConfigInterface.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\cloudflare\CloudflareConfigInterface.
+ */
+
+namespace Drupal\cloudflare;
+
+/**
+ * Zone methods for CloudFlare.
+ */
+interface CloudFlareConfigInterface {
```

Comment:
Yeah I am glad you picked up on this too.  The main issue here is I need to be able to easily support multiple zones while also hiding those details from the 99% that couldn't care less because they only have 1 zone.